ouroboros-consensus-cardano/src/tools/Cardano/Tools/DBTruncater/Types.hs

```diff
@@ -0,0 +1,14 @@
+module Cardano.Tools.DBTruncater.Types (module Cardano.Tools.DBTruncater.Types) where
+
+import           Cardano.Tools.DBAnalyser.Types
+import           Ouroboros.Consensus.Block.Abstract
+
+data DBTruncaterConfig = DBTruncaterConfig {
+    dbDir         :: FilePath
+  , truncatePoint :: TruncatePoint
+  , blockType     :: BlockType
+  , verbose       :: Bool
+  }
+
+newtype TruncatePoint = TruncatePoint SlotNo
```

Comment:
I think this would be a good place to add some Haddock that clarifies whether the requested truncation slot is inclusive or exclusive.